### PR TITLE
feat(vrack-services): add express order

### PR DESCRIPTION
**Pull request by chipp972**
ref: MANAGER-13528

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-13528
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

- Implemented express order on vRack Services
- Created a new module `order`
- Moved the generic helpers for msw and playwright
- Moved the order mocks in the module
- Updated the tests

### CODE REVIEW

**Review by aboungnaseng-ovhcloud — APPROVED** [review]

File: packages/manager/apps/vrack-services/e2e/utils/network.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { BrowserContext } from '@playwright/test';
+import { ICustomWorld } from '@playwright-helpers';
+import { toPlaywrightMockHandler } from '../../../../../../playwright-helpers';
```
Comment (ghyenne) [inline packages/manager/apps/vrack-services/e2e/utils/network.ts:3]:
```suggestion
import { ICustomWorld, toPlaywrightMockHandler } from '@playwright-helpers';
```
If it's same path / alias for import

File: packages/manager/modules/order/mock/order-details.ts [hunk 2]
```diff
@@ -1,7 +1,6 @@
-import { PathParams } from 'msw';
-import { getParamsFromUrl } from '../../e2e/utils/playwright-helpers';
-import { Handler } from '../../e2e/utils/msw-helpers';
-import { OrderData, OrderDetail } from '@/api/order';
+import { Handler } from '../../../../../playwright-helpers';
```
Comment (ghyenne) [inline packages/manager/modules/order/mock/order-details.ts:1]:
```suggestion
import { Handler } from '@playwright-helpers';
```

File: packages/manager/modules/order/mock/order-details.ts [hunk 3]
```diff
@@ -1,7 +1,6 @@
...
+import { OrderData, OrderDetail } from '../src';
+import { getParamsFromUrl } from '../../../../../playwright-helpers/network';
```
Comment (ghyenne) [inline packages/manager/modules/order/mock/order-details.ts:3]:
```suggestion
import { getParamsFromUrl } from '@playwright-helpers/network';
```

File: packages/manager/modules/order/mock/cart.ts [hunk 4]
```diff
@@ -1,7 +1,6 @@
-import { PathParams } from 'msw';
-import { Handler } from '../../e2e/utils/msw-helpers';
-import { getParamsFromUrl } from '../../e2e/utils/playwright-helpers';
-import { Cart, Item } from '@/api/order/order.type';
+import { Handler } from '../../../../../playwright-helpers';
```
Comment (ghyenne) [inline packages/manager/modules/order/mock/cart.ts:1]:
```suggestion
import { Handler } from '@playwright-helpers';
```

File: packages/manager/modules/order/mock/cart.ts [hunk 5]
```diff
@@ -1,7 +1,6 @@
...
+import { Cart, Item } from '../src';
+import { getParamsFromUrl } from '../../../../../playwright-helpers/network';
```
Comment (ghyenne) [inline packages/manager/modules/order/mock/cart.ts:3]:
```suggestion
import { getParamsFromUrl } from '@playwright-helpers/network';
```

File: packages/manager/apps/vrack-services/src/index.tsx [hunk 6]
```diff
@@ -5,7 +5,8 @@ import {
   ShellContext,
   initShellContext,
 } from '@ovh-ux/manager-react-shell-client';
-import { getMswHandlers } from '../mock/handlers';
+import { toMswHandlers } from '../../../../../playwright-helpers/msw';
```
Comment (ghyenne) [inline packages/manager/apps/vrack-services/src/index.tsx:8]:
```suggestion
import { toMswHandlers } from '@playwright-helpers/msw';
```

File: packages/manager/apps/vrack-services/mock/vrack/association.ts [hunk 7]
```diff
@@ -1,8 +1,7 @@
-/* eslint-disable import/no-extraneous-dependencies */
 import { PathParams } from 'msw';
-import { Request as PlaywrightRequest } from 'playwright';
-import { getParamsFromUrl } from '../../e2e/utils/playwright-helpers';
-import { Handler } from '../../e2e/utils/msw-helpers';
+import { Request as PlaywrightRequest } from '@playwright/test';
+import { Handler } from '@playwright-helpers';
+import { getParamsFromUrl } from '../../../../../../playwright-helpers/network';
```
Comment (ghyenne) [inline packages/manager/apps/vrack-services/mock/vrack/association.ts:4]:
```suggestion
import { getParamsFromUrl } from '@playwright-helpers/network';
```

File: packages/manager/apps/vrack-services/mock/vrack-services/vrack-services.ts [hunk 8]
```diff
@@ -1,7 +1,7 @@
-/* eslint-disable import/no-extraneous-dependencies */
 import { PathParams } from 'msw';
-import { Request as PlaywrightRequest } from 'playwright';
-import { getParamsFromUrl, Handler } from '../../e2e/utils';
+import { Request as PlaywrightRequest } from '@playwright/test';
+import { Handler } from '@playwright-helpers';
+import { getParamsFromUrl } from '../../../../../../playwright-helpers/network';
```
Comment (ghyenne) [inline packages/manager/apps/vrack-services/mock/vrack-services/vrack-services.ts:4]:
```suggestion
import { getParamsFromUrl } from '@playwright-helpers/network';
```

File: packages/manager/apps/vrack-services/e2e/utils/network.ts (hunk 1 above)
Comment (chipp972) [reply packages/manager/apps/vrack-services/e2e/utils/network.ts:3]: aliases don't work in playwright

File: packages/manager/modules/order/mock/order-details.ts (hunk 2 above)
Comment (chipp972) [reply packages/manager/modules/order/mock/order-details.ts:1]: the alias is not configured in the order package and it doesn't work well in playwright anyways

File: packages/manager/modules/order/mock/order-details.ts (hunk 3 above)
Comment (chipp972) [reply packages/manager/modules/order/mock/order-details.ts:3]: aliases don't work in playwright

File: packages/manager/modules/order/mock/cart.ts (hunk 4 above)
Comment (chipp972) [reply packages/manager/modules/order/mock/cart.ts:1]: aliases don't work in playwright

File: packages/manager/modules/order/mock/cart.ts (hunk 5 above)
Comment (chipp972) [reply packages/manager/modules/order/mock/cart.ts:3]: aliases don't work in playwright

File: packages/manager/apps/vrack-services/mock/vrack/association.ts (hunk 7 above)
Comment (chipp972) [reply packages/manager/apps/vrack-services/mock/vrack/association.ts:4]: aliases don't work in playwright

**Pull request merged**